### DISCUSSION TITLE
OptionMenu is tk and not ttk

### DISCUSSION THREAD

**Discussion by AJ6CU** [alejandroautalan/pygubu-designer#177] [Q&A] [answered]
Hi,

Noticed that selecting an OptionMenu I get class tk.OptionMenu instead of ttk.OptionMenu. Noticed this because I was trying to use the set_menu function of ttk.OptionMenu. Flipped the tk and ttk switch, always tk.OptionMenu.

Regards,

Mark

**Comment by alejandroautalan** [comment, marked as answer]
Hello @AJ6CU thanks for your comment.

Well, you found a bug. In the beginnings of pygubu there was no OptionMenu in the ttk module, so the tk.OptionMenu was in both widget palettes. I did not  noticed that was added to the ttk module.

I will fix this ASAP.

Regards!
Alejandro A.

**Comment by AJ6CU** [comment]
Thanks for the acknowledgement.  I switched to a ComboBox and have a simple solution now to update the menu items. So no rush to fix from this side!

 

Regards,

 

Mark
